cmsplugin_articles_ai/factories.py

```diff
@@ -0,0 +1,93 @@
+# -*- coding: utf-8 -*-
+import random
+import string
+from datetime import timedelta
+
+import factory
```

Comment:
It might be a good idea to move the tests and factories to another package.
That way factory-boy becomes a dev-only dependency and we don't end up installing the tests in virtualenvs.